modules/ROOT/partials/events/autocorrect-events.adoc

```diff
@@ -0,0 +1,10 @@
+The following events are provided by the xref:autocorrect.adoc[Spelling Autocorrect] plugin.
+
+[cols="1,1,2",options="header"]
+|===
+|Name |Data |Description
+|AutocorrectStart |N/A |Fired when autocorrect is turned on after being off.
```

Comment:
Do we need to mention whether it is fired by the initial state of config/localStorage?

Reply:
Not sure. I think we should simplify the description similar to the description of the spellchecker events.  

```
|SpellcheckStart |N/A |Fired when spellchecking is enabled.
|SpellcheckEnd |N/A |Fired when spellchecking is disabled.
|SpellcheckerLanguageChanged |`+{ language: string, prevLanguage: string }+` |Fired when the spellchecking language is changed.
```

If we should mention that the autocorrect events are fired by the initial state of the config, I think we should do the same for `SpellCheckerLanguageChagned`

Reply:
This unfinished discussion is now [DOC-1800](https://ephocks.atlassian.net/browse/DOC-1800).